test/dependency_source/test_requirement.py

```diff
@@ -378,7 +379,9 @@ def test_requirement_source_fix_rollback_failure(monkeypatch, req_file):
     def mock_replace(*_args, **_kwargs):
         raise OSError
 
-    monkeypatch.setattr(os, "replace", mock_replace)
+    from tempfile import _TemporaryFileWrapper
+
+    monkeypatch.setattr(_TemporaryFileWrapper, "seek", mock_replace, raising=False)
```

Comment:
This seems slightly off: we're replacing `seek` with a mocked `os.replace` implementation, it looks like.

Reply:
I agree, it's not ideal, but as it is related to [this part](https://github.com/pypa/pip-audit/pull/757/files#diff-a05f9f5ebfa6898a1b210166fc630ca5558f90eb0b6358267f66288f27bbdc8aL281-R282), which is also new, it's the only way I found to keep the test behavior intact

Reply:
I understand that! I'm saying that it should be renamed to `mock_seek` since that's what it's mocking 😉

Reply:
oh yes, sorry, I didn't realize you were talking about the variable name 😅 I changed it.